### DISCUSSION TITLE
add to clipboard action?

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#4480] [Ideas]
Hello,

It would be nice to have an add to clipboard action.

In the meantime, how can I do have such a behavior?

Thanks a lot,

**Comment by weixingp** [comment]
For anyone looking for a quick workaround:
Using the embed component:
`<button onclick="(
function() { 
navigator.clipboard.writeText('{{ value }}').then(function (){alert ('Copied!')})
}
)()">Copy Link</button>`

You can then add some CSS to style the button

**Comment by mariohof** [comment]
Please, add a "Copy" button to all code blocks in Markdown Viewer.

**Comment by immohacker** [comment]
Hi, is this function not vaild anymore?
ive got en Java script error.
have you an idea?
Or exist an other solution?
Thank you!
Best regards Bonkers

**Comment by immohacker** [comment]
Sorry solved my self.
Iam stupid.
Because its sounds at JavaScript, i inserted in javascript.
Its HTMl text of course
Sorry for this.
Best regards Bonkersdeluxe
